scripts/fedbiomed_run

```diff
@@ -271,11 +268,12 @@ case $1 in
             export PARAMS=
         fi
 
-        # TODO : better test with python module loading attempt and try/except
-#        source ${basedir}/scripts/fedbiomed_environment node "$IP_ADDRESS" -v
+
+         # Activate node envirenment
+        source "$basedir"/scripts/fedbiomed_environment node "$IP_ADDRESS"
 
         # silently initialize environment before any command
-        if ! forward_command_to_python_cli node "configuration create" "$IP_ADDRESS" >/dev/null; then
+        if ! python -m fedbiomed.common.cli configuration create --component node --name $CONFIG_FILE -uc  >/dev/null; then
```

Comment:
A (minor ?) issue with new syntax is that this results in creating 2 distinct node configurations:

```
./scripts/fedbiomed_run config config-n2.ini configuration create -c node -n config-n3.ini
```

This is a bit misleading (annoyance) but not really a bug, we might choose to ignore.

But I don't see any obvious way to improve while we are in this mixed handling of parameters between fedbiomed_run and CLI parser

Reply:
No actually, I forgot to remove `initialize_create_configuration` from `node.cli` and `researhcer.cli`. However, -which is also related to comment above- if configuration is created with component specific CLI it will create two distinct configuration.

Reply:
Would you push that modif without the `initialize_create_configuration` then ?
This might open room for a nicer solution to the environ/config issue :-)

Reply:
Yes I am going to push that version soon.

Reply:
I pushed the new version a moment ago.